samplesvc/src/main/scala/org/squbs/sample/sample/svc/ShutdownHook.scala

```diff
@@ -0,0 +1,16 @@
+package org.squbs.sample.sample.svc
+
+import org.squbs.lifecycle.{ExtensionLifecycle, GracefulStop}
+import org.squbs.unicomplex.Unicomplex
+
+class ShutdownHook extends ExtensionLifecycle {
+  override def init(): Unit = {
+    Runtime.getRuntime.addShutdownHook(new Thread() {
+      override def run() {
+        Unicomplex(boot.actorSystem).uniActor ! GracefulStop
+        System.out.println("Shutdown Hook Triggered")
```

Comment:
Should we move this to the line before the `GracefulStop` message?  Just ordering wise, the logs would make better sense?